src/UnitTests/Internationalization.cs

```diff
@@ -16,12 +16,12 @@ public class Order
 
             public class Customer
             {
-                public string ��� { get; set; }
+                public string Æøå { get; set; }
```

Comment:
The encoding was wrong on these characters. They were encoded in "Western European (Windows)", but the file is encoded in UTF-8. I can only guess that on Windows, some kind of encoding munging is done to allow it pass through the compiler, but on Linux no such "convenience" is present, so compilation was failing.